torch/_functorch/partitioners.py

```diff
@@ -1403,11 +1405,28 @@ def _optimize_runtime_with_given_memory(
         return ilp_knapsack(memory, runtimes, max_memory)
     elif SOLVER == "dp":
         return dp_knapsack(memory, runtimes, max_memory)
-    elif callable(SOLVER):
-        saved_node_idx, recomp_node_idx = SOLVER(
-            memory, joint_graph, max_memory, node_info, all_recomputable_banned_nodes
```

Comment:
Let's keep this around.  It still seems like a useful feature for others.

Reply:
Updating to bring this back 👍